content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -65,6 +68,28 @@ Otherwise, it's placed in the internal unschedulable Pods list, and doesn't get
 For more details about how internal scheduler queues work, read
 [Scheduling queue in kube-scheduler](https://github.com/kubernetes/community/blob/f03b6d5692bd979f07dd472e7b6836b2dad0fd9b/contributors/devel/sig-scheduling/scheduler_queues.md).
 
+### EnqueueExtension
+
+EnqueueExtension is the interface where the plugin can control 
+whether to retry scheduling of Pods rejected by this plugin, based on changes in the cluster.
+Plugins that implement PreEnqueue, PreFilter, Filter, Reserve or Permit should implement this interface.
+
+#### QueueingHint
+
+{{< feature-state for_k8s_version="v1.28" state="beta" >}}
+
+QueueingHint is a callback function for deciding whether a Pod can be requeued to the active queue or backoff queue. 
+It's executed every time a certain kind of events happen,
+and when the QueueingHint finds that the event might make the Pod schedulable, 
+the Pod is put into the active queue or the backoff queue
+so that the scheduler will retry the scheduling of the Pod.
+
+{{< note >}}
+QueueingHint is a beta-level field and is enabled by default in 1.28. 
```

Comment:
```suggestion
QueueingHint is a beta-level feature and is enabled by default in 1.28. 
```
(it's not an API field)

Reply:
This isn't quite right either. Try something like “Queuing hint evaluation during scheduling is a beta-level feature&hellip;”, or “the QueueingHint scheduler frobnication is a beta feature&hellip;” (change frobnication to a suitable term).

Reply:
I go with:
QueueingHint evaluation during scheduling is a beta-level feature ..